src/PeaceComponents/Charts/Orderbychat.tsx

```
@@ -110,14 +110,14 @@ export default function OrderByChat() {
 
         switch (SelectedPeriod) {
             case "7 days":
-                const weekAgo = new Date();
-                weekAgo.setDate(now.getDate() - 7);
-                return alldata.filter(entry => new Date(entry.date) >= weekAgo);
+                const weekago = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)